examples/consistentE/basicE.py

```diff
@@ -31,7 +31,7 @@
 ds = DREAMSettings()
 
 # set collision settings
-ds.collisions.collfreq_mode = Collisions.COLLFREQ_MODE_FULL
+ds.collisions.collfreq_mode = Collisions.COLLFREQ_MODE_SUPERTHERMAL
```

Comment:
Yeah, I didn't intend to keep this change. But it doesn't seem to work either way, with COLLFREQ_MODE_FULL the solver does not converge, and with COLLFREQ_MODE_SUPERTHERMAL it hangs before the time stepping even starts...

Reply:
Oh, wait, it runs if I also change the temperature back. But I am still a bit confused about this example, the description and comments do not seem to agree with the actual settings

Reply:
Interesting... Well, this was the very first simulation we ever set up, and for a long time it served as our go-to test script (with tons of modifications to test out new features as a result). If you've managed to find a set of parameters which seem to work, then it would be great if you can commit them (come to think of it, I think some new DREAM users have pointed out that they tested this script without success). Otherwise, no need to spend any time on fixing this now.